docs/cas-server-documentation/logging/Logging-MDC.md

```diff
@@ -36,6 +36,9 @@ may convey additional information about the nature of the request or the authent
 | `principal`                         | CAS authenticated principal id.
 
 Additionally, all available request attributes, headers, and parameters are exposed as variables.
+<div class="alert alert-warning">
+  <strong>Pay Attention</strong><br /> These variables potentially include the password. If you set `includeMDC=true` in [syslog appender](Logging-Syslog.html), these informations, including clear password, will be sent to syslog.
```

Comment:
This is rendered as HTML, not markdown. You should be using HTML tags for links and for `<code>`

Reply:
I've made another commit to change the link. Sorry for the multiple mistakes. This will at least make me learn.